src/routes/PostDetail/Comments/Comment/__snapshots__/Comment.test.js.snap

```diff
@@ -21,7 +21,7 @@ exports[`Comment displays image attachments 1`] = `
     <span
       data-stylename="timestamp"
     >
-      2 mos ago
+      3 mos ago
```

Comment:
ugh this is so annoying, gotta figure out how to not have this happen someday...